packages/gestalt/src/SearchField.js

```diff
@@ -26,49 +26,32 @@ type Props = {|
   value?: string,
 |};
 
-type State = {|
-  focused: boolean,
-  hovered: boolean,
-|};
-
-export default class SearchField extends React.Component<Props, State> {
-  static propTypes = {
-    accessibilityLabel: PropTypes.string.isRequired,
-    autoComplete: PropTypes.oneOf(['on', 'off', 'username', 'name']),
-    id: PropTypes.string.isRequired,
-    onBlur: PropTypes.func,
-    onChange: PropTypes.func.isRequired,
-    onFocus: PropTypes.func,
-    placeholder: PropTypes.string,
-    size: PropTypes.oneOf(['md', 'lg']),
-    value: PropTypes.string,
-  };
-
-  state = {
-    focused: false,
-    hovered: false,
-  };
-
-  handleChange = (event: SyntheticEvent<HTMLInputElement>) => {
-    const { onChange } = this.props;
+const SearchField = ({
+  accessibilityLabel,
+  autoComplete,
+  id,
+  onBlur,
+  onChange,
+  onFocus,
+  placeholder,
+  size = 'md',
+  value,
+}: Props) => {
+  const [focused, setFocused] = useState(false);
+
+  const handleChange = (event: SyntheticEvent<HTMLInputElement>) => {
     onChange({
       value: event.currentTarget.value,
       syntheticEvent: event,
     });
   };
 
-  handleClear = (event: SyntheticEvent<HTMLInputElement>) => {
-    const { onChange } = this.props;
+  const handleClear = (event: SyntheticEvent<HTMLInputElement>) => {
     onChange({ value: '', syntheticEvent: event });
   };
 
-  handleMouseEnter = () => this.setState({ hovered: true });
```

Comment:
@nachofassini I think we do need a hover state, check out `TextField` and the Figma designs. The border color changes from `#ddd` to `#d0d0d0` when hovering.

Reply:
Fixed!